packages/config/config/devices/0x0431/ecodim.json

```
@@ -78,13 +83,17 @@
 					"value": 0
 				},
 				{
-					"label": "Send switch multilevel report",
+					"label": "Send Switch Multilevel Report",
 					"value": 1
+				},
+				{
+					"label": "Send Switch Multilevel Report only when Load status is not changed by Z-WAVE Command",
+					"value": 2
 				}
 			]
 		},
 		"3": {
-			"label": "Dimming speed",
+			"label": "Default dimming speed",
 			"description": "Default dimming speed，unit is second. Valid values: 0-100. Default value: 5 0 - Instanly. 1 - Fast. ... 100 - Slow.",
```

```suggestion
			"description": "Default dimming speed",
```
but that is rather pointless

---

Yes agree, but it's from the manual